cmd/buildah/push.go

```diff
@@ -79,10 +93,16 @@ func pushCmd(c *cli.Context) error {
 		return err
 	}
 
+	systemContext, err := systemContextFromOptions(c)
+	if err != nil {
+		return errors.Errorf("error building system context [%v]", err)
```

Comment:
Change this to return `errors.Wrapf(err, "error building system context")`.

Reply:
Fixed.  Circled back and changed in from.go too.